pkg/version/version_test.go

```diff
@@ -114,7 +114,7 @@ func TestGetVersionsGithub(t *testing.T) {
 	t.Run("error on bad addr", func(t *testing.T) {
 		version, err := GetLatestReleaseGithub("http://a.super.non.existant.domain/")
 		assert.Equal(t, "", version)
-		assert.EqualError(t, err, "Get \"http://a.super.non.existant.domain/\": dial tcp: lookup a.super.non.existant.domain: no such host")
+		assert.Containsf(t, err.Error(), "Get \"http://a.super.non.existant.domain/\": dial tcp: lookup a.super.non.existant.domain", ": no such host")
```

Comment:
I run the test and got the error message "dial tcp: lookup a.super.non.existant.domain on 192.168.0.1:53: no such host" so change the assert condition.